src/features/congregation/settings/language_groups/group_add/useGroupAdd.tsx

```
@@ -1,102 +1,104 @@
     try {
-      const appSettings = convertSettingsObjectToArray(
-        structuredClone(settings)
-      );
+      const groups = structuredClone(allGroups);
 
-      const languageGroups = appSettings.cong_settings.language_groups.groups;
-
-      const findGroup = languageGroups.find(
-        (record) => record.name === group.name
+      const findGroup = groups.find(
+        (record) => record.group_data.name === record.group_data.name
```

Biome (1.9.4)

[error] Comparing to itself is potentially pointless.



(lint/suspicious/noSelfCompare)